README.md

```diff
@@ -27,146 +27,85 @@ https://unpkg.com/mmdynamo@0.2.0/dist/mmdynamo.cjs.js
 ## Initializing dynamodb
 
 ```js
-import { DynamoDB } from "mmdynamo";
-
-const client = new DynamoDB({
-  region: "us-east-2",
-  accessKeyId: "",
-  secretAccessKey: "",
-  service: "dynamodb",
-  endpoint: "http://dynamodb.us-east-2.amazonaws.com",
+import  Client  from "mmdynamo";
+
+const client = new Client({
+  agent: "fetch",
+  apiKey: "xxxx",
+  federationUrl: "http://xxx.xxx.macrometa.io",
+  // this is an optional parameter to be passed when you want to use the endpoint as it is
```

Comment:
put this inline with `absoluetPath:true`.

Reply:
done

Reply:
and put space after `:`

Reply:
done